test/_lib/integration-02-local-component1/package.json

```diff
@@ -13,10 +13,7 @@
         },
         "remoteComponent3": {
           "version": "^2.0.0",
-          "methods": {
-            "method1": {},
-            "method2": {}
-          }
+          "discoverMethods": true
```

Comment:
This looks new, probably need to add it to the documentation

Reply:
Absolutely - but I just want to write some more tests in happner-client